src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -1251,6 +1251,58 @@ public void AnalyzeCommandBase_ShouldOnlyLogArtifactsWhenResultsAreFound()
             }
         }
 
+        [Fact]
+        public void AnalyzeCommandBase_ShouldNotThrowException_WhenAnalyzingSameFileBasedOnTwoTargetFileSpecifiers()
+        {
+            var files = new List<string>
+            {
+                $@"{Environment.CurrentDirectory}\Error.dll"
+            };
+
+            var testCases = new[]
+            {
+                new
+                {
+                    IsMultithreaded = false
+                },
+                new
+                {
+                    IsMultithreaded = true
+                }
+            };
+
+            Action action = () =>
+            {
+                foreach (var testCase in testCases)
```

Comment:
>foreach (var testCase in testCases) [](http://example.com/codeflow?start=16&length=35)

foreach (bool multithreaded in new bool[] { true, false}

Reply:
done!
thanks for the suggestion!